groupsforsections.py

```diff
@@ -207,45 +208,54 @@ def main():
     utils.setup_logging()
     logging.info('Script Started')
     logging.debug('args: ' + str(sys.argv))
+    # if the environmental variables are not present then we are making an assumption here that the script is running
+    # locally not in Open shift
+    if "COURSE_ID" not in environ or "CATEGORY_NAME" not in environ :
```

Comment:
FYI:  This could be written as `set(('COURSE_ID', 'CATEGORY_NAME')).issubset(environ)`